views/index-getInvolved.html

```diff
@@ -0,0 +1,92 @@
+      </div>
+      <div class="col-sm-10 col-xs-12 involvedDescription">
+        <p>Our Facebook group is the place to connect with fellow campers in the Indianapolis area. Stay up to date on activities, ask for help on a coding project, show a blog post, etc. </p>
+      </div>
+      </div>
+    </div>
+  </div>
+  <div class="panel panel-default">
+    <div class="panel-heading" role="tab" id="headingTwo" data-toggle="collapse" data-parent="#accordion" href="#collapseTwo" aria-expanded="true" aria-controls="collapseTwo">
+      <h4 class="panel-title">
+        <a class="collapsed" role="button">
+          Come to a Meetup
+        </a>
+      </h4>
+    </div>
+    <div id="collapseTwo" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingTwo">
+      <div class="panel-body">
+        <div class="col-sm-2 col-sm-offset-0 col-xs-4 col-xs-offset-4 involvedImage"><a href="https://www.meetup.com/Free-Code-Camp-Indy/" target="_blank"><img src="/images/get_involved/meetup.png"></a></div>
+      <div class="col-sm-10 col-xs-12 involvedDescription">
+        <p>Join our Meetup Group and come to a meeting. We hold workshops, hackathons, talks and more. Everyone from beginners to experts are welcome.</p>
+      </div>
+      </div>
+    </div>
+  </div>
+  <div class="panel panel-default">
+    <div class="panel-heading" role="tab" id="headingThree" data-toggle="collapse" data-parent="#accordion" href="#collapseThree" aria-expanded="true" aria-controls="collapseThree">
+      <h4 class="panel-title">
+        <a class="collapsed" role="button">
+          Start or Contribute to a Project
+        </a>
+      </h4>
+    </div>
+    <div id="collapseThree" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingThree">
+      <div class="panel-body">
+        <div class="col-sm-2 col-sm-offset-0 col-xs-4 col-xs-offset-4 involvedImage"><a href="#" target="_blank"><img src="/images/get_involved/github.png"></a></div>
+      <div class="col-sm-10 col-xs-12 involvedDescription">
+        <p>Check out our Project Page. If you see an open project that interests you, start contributing. Or start your own project.</p>
+      </div>
+      </div>
+    </div>
+  </div>
+  <div class="panel panel-default">
+    <div class="panel-heading" role="tab" id="headingFour" data-toggle="collapse" data-parent="#accordion" href="#collapseFour" aria-expanded="false" aria-controls="collapseFour">
+      <h4 class="panel-title">
+        <a class="collapsed" role="button">
+          Add your Blog
+        </a>
+      </h4>
+    </div>
+    <div id="collapseFour" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingFour">
+      <div class="panel-body">
+        <div class="col-sm-2 col-sm-offset-0 col-xs-4 col-xs-offset-4 involvedImage"><a href="#" target="_blank"><img src="http://res.cloudinary.com/dpyptkefp/image/upload/c_scale,w_120/v1476713784/fb_icon_325x325_r17eed.png"></a></div>
```

Comment:
`images/get_involved/facebook.png`

`https://www.facebook.com/groups/free.code.camp.indianapolis.in`